src/devices/ubisys.ts

```diff
@@ -725,6 +726,54 @@ const definitions: Definition[] = [
         },
         ota: ota.ubisys,
     },
+    {
+        // S1-R Series 2 uses the same modelId as the regular S1-R, but the energy clusters are located in endpoint 1 (instead of 4, like the regular S1-R).
+        fingerprint: [
+            {
+                manufacturerName: 'ubisys',
+                modelID: 'S1-R (5601)',
+                endpoints: [{ID: 1, profileID: 260, deviceID: 266, inputClusters: [0, 3, 4, 5, 6, 1794, 2820], outputClusters: []}],
+            },
+        ],
+        model: 'S1-R (Series 2)',
```

Comment:
```suggestion
        model: 'S1-R-2',
```

Can we use this here? It's a bit more friendly without the spaces

Reply:
done